README.md

```diff
@@ -252,51 +252,26 @@ To utilize the `span` option above you need to pass an opentracing span into the
 example shows you how to setup an express app to track requests and any calls made via a wag client.
```

Comment:
i might also throw in that if you want to guarantee that a trace is sampled you can set it in the request.span object after the middleware is run like this:
```
req.span.setTag("sampling.priority", 1)
```